### PR TITLE
file name typo

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
Fixed typo in file name.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -154,7 +154,7 @@ test/gui: run-cmake-debug
 	$(XVFB) ./dbuild/bin/texteditor --replay tests/TestGui/gui_text_editor.tcl
 	$(XVFB) ./dbuild/bin/newfile --replay tests/TestGui/gui_new_file.tcl
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_foedag.tcl
-	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_foedag_negetive.tcl && exit 1 || (echo "PASSED: Caught negative test")
+	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_foedag_negetive_test.tcl && exit 1 || (echo "PASSED: Caught negative test")
```
Comment (ravic-rs) [inline Makefile:157]:
```suggestion
	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_foedag_negative_test.tcl && exit 1 || (echo "PASSED: Caught negative test")
```

**Review by ravic-rs — COMMENTED** [review]
Minor suggestion since we are fixing the typo.

File: Makefile (hunk 1 above)
Comment (KochynVolodymyr) [reply Makefile]: Done

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**